metrics/web/docs/features/public_dashboard/diagrams/public_dashboard_sequence_diagram.puml

```diff
@@ -0,0 +1,60 @@
+@startuml public_dashboard_sequence_diagram
+
+skinparam ParticipantPadding 20
+skinparam BoxPadding 10
+
+actor Bob
+
+participant Browser
+participant LoadingPage
+participant AuthNotifier
+participant FeatureConfigNotifier
+participant NavigationNotifier
+participant CoolPage
+participant LoginPage
+
+Bob -> Browser ++:  opens a "Cool Page" URL
+Browser -> LoadingPage ++: navigates to initialize all components
+LoadingPage --> Browser: returns the LoadingPage UI
+Browser --> Bob: sees the LoadingPage UI
+
+LoadingPage -> AuthNotifier ++: addListener(_authNotifierListener)
+AuthNotifier --> LoadingPage --: the listener is added
+LoadingPage -> AuthNotifier ++: subscribeToAuthenticationUpdates()
+AuthNotifier --> LoadingPage --: subscribed to auth updates
+
+LoadingPage -> FeatureConfigNotifier ++: addListener(_featureConfigNotifierListener)
+FeatureConfigNotifier --> LoadingPage --: the listener is added
+LoadingPage -> FeatureConfigNotifier ++: initializeConfig()
+FeatureConfigNotifier --> LoadingPage --: the config initialization is started
+
+AuthNotifier --> AuthNotifier ++: receives the user state
+AuthNotifier -> NavigationNotifier ++: handleAuthUpdates(authState)
+NavigationNotifier --> AuthNotifier --: the auth updates are handled
+AuthNotifier --
+
+FeatureConfigNotifier --> FeatureConfigNotifier ++: receives the feature config
+FeatureConfigNotifier --> LoadingPage: the feature config is initialized
+FeatureConfigNotifier --> AuthNotifier --: handlePublicDashboardFeatureConfigUpdates(model)
```

Comment:
🔨 
It seems like `FeatureConfigNotifier` does not calls this method directly. It seems like `InjectionContainer` makes this call.